### DISCUSSION TITLE
Fillna /mean

### DISCUSSION THREAD

**Discussion by sabdul4** [AammarTufail/python_ka_chilla2023#85] [Q&A]
![missi](https://user-images.githubusercontent.com/117725522/213907165-b42e7eb8-90cb-4094-aae6-4e4fe594f625.jpg)
fillna kr rha hn mean k sath ye error aa rha

**Comment by nouman073** [comment]
I think spelling mistake in the column name

**Reply by sabdul4** [reply]
![image](https://user-images.githubusercontent.com/117725522/213925294-18326900-f822-498f-ac3e-ee2a244068f3.jpeg)
Ni spelling mistake ni h blke pehle coulmn k name m msla aa rha tha q k bracket () the then mene mene change kr k dosra nam diya
